doc/reuse/links.txt

```diff
@@ -0,0 +1,4 @@
+.. _reStructuredText style guide: https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/
+.. _Read the Docs at Canonical: https://library.canonical.com/documentation/read-the-docs
+.. _How to publish documentation on Read the Docs: https://library.canonical.com/documentation/publish-on-read-the-docs
```

Comment:
I presume library.canonical.com is not accessible to community contributors, is that correct?

Reply:
Goog point. I'll remove those.